.env.example

```diff
@@ -60,8 +60,14 @@ HEURISTAIAPIKEY     = '<add_your_heuristai_api_key_here>'
 # ANTHROPIC_API_KEY   = '<add_your_anthropic_api_key_here>'
 
 # Front end container details
-VITE_JSON_RPC_SERVER_URL  = 'http://127.0.0.1:4000/api'
-VITE_WS_SERVER_URL   = 'ws://127.0.0.1:4000'
+VITE_JSON_RPC_SERVER_URL  = 'http://127.0.0.1:4000/api' # if VITE_PROXY_ENABLED = 'true' change to your jsonrpc server url (/api)
+VITE_WS_SERVER_URL   = 'ws://127.0.0.1:4000' # if VITE_PROXY_ENABLED = 'true' change to your websocket server url (/socket.io)
 VITE_PLAUSIBLE_DOMAIN = 'simulator.genlayer.com'
 FRONTEND_PORT   = '8080'
+
+VITE_PROXY_ENABLED = 'false'
+VITE_PROXY_JSON_RPC_SERVER_URL = 'http://127.0.0.1:4000/api' # if VITE_PROXY_ENABLED = 'true' change to your jsonrpc server url (http://jsonrpc:4000/api)
+VITE_PROXY_WS_SERVER_URL = 'ws://127.0.0.1:4000' # if VITE_PROXY_ENABLED = 'true' change to your websocket server url (ws://jsonrpc:4000)
```

Comment:
I think it'd be best to default these to `http://jsonrpc:4000/api` and `ws://jsonrpc:4000`

Reply:
updated the env